src/bloom.h

```diff
@@ -103,7 +104,11 @@ class CBloomFilter
 
 /**
  * RollingBloomFilter is a probabilistic "keep track of most recently inserted" set.
- * Construct it with the number of items to keep track of, and a false-positive rate.
+ * Construct it with the number of items to keep track of, and a false-positive
+ * rate. Unlike CBloomFilter, by default nTweak is set to a cryptographically
+ * secure random value for you. Similarly rather than clear() the method
```

Comment:
I'm not saying this is a problem, only that calling a random 32-bit number cryptographic is a bit grey area. I would just call it secure, because something of cryptographic strength isn't necessary here.